From 06cc8e469088cc881c3cdc14685a0fe297e9cd3e Mon Sep 17 00:00:00 2001
From: Manish Narani <manish.narani@xilinx.com>
Date: Fri, 14 Feb 2020 00:53:05 -0700
Subject: [PATCH 1195/1852] dt-bindings: Update binding for pinctrl in arasan
 SD driver

commit e2a532566ba0fe8eb805cce2480c25e44699d4a4 from
https://github.com/Xilinx/linux-xlnx.git

Add optional pinctrl description to binding document.

Signed-off-by: Manish Narani <manish.narani@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/mmc/arasan,sdhci.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/mmc/arasan,sdhci.txt b/Documentation/devicetree/bindings/mmc/arasan,sdhci.txt
index 404b0ab48bad..75c955f16066 100644
--- a/Documentation/devicetree/bindings/mmc/arasan,sdhci.txt
+++ b/Documentation/devicetree/bindings/mmc/arasan,sdhci.txt
@@ -52,6 +52,8 @@ Optional Properties:
     properly. Test mode can be used to force the controller to function.
   - xlnx,int-clock-stable-broken: when present, the controller always reports
     that the internal clock is stable even when it is not.
+  - pinctrl-0: pin control group to be used for this controller.
+  - pinctrl-names: must contain a "default" entry.
 
   - xlnx,mio-bank: When specified, this will indicate the MIO bank number in
     which the command and data lines are configured. If not specified, driver
-- 
2.31.1

